### PR TITLE
TLC- 258 create Side Panel PO

### PR DESCRIPTION
**Pull request by cyxoi1**

### CODE REVIEW

File: pageObjects/sidePanel.page.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import BasePage from './Base.page';
+import { RegisterData } from '../20608/data/DataPage';
```
Comment (vasylynash) [inline pageObjects/sidePanel.page.js:2]: I'm not sure it's ok to use your group data in general PO

File: pageObjects/sidePanel.page.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  deletePhone() {
+    this.settingsEditPhone.clearValue();
+    this.saveBtn.click();
+  }
+
+  moveToSettings() {
+    this.openDropdown.click();
+    this.settings.click();
```
[the remark this answers is not in the document]
Comment (GalinaRuban) [reply pageObjects/sidePanel.page.js:82]: I also agree that it's not OK, even though it's not the first group that did it

File: pageObjects/sidePanel.page.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    return $('[class="ant-btn ant-btn-primary"]');
+  }
+
+  get openDropdown() {
+    return $('[class="ant-dropdown-link ant-dropdown-trigger"]');
+  }
+
+  get settings() {
+    return $('//a[contains(text(),"Settings")]');
+  }
```
[the remark this answers is not in the document]
Comment (cyxoi1) [reply pageObjects/sidePanel.page.js:58]: I needed for my method and couldn't find it in PO files at that moment

File: pageObjects/sidePanel.page.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (cyxoi1) [reply pageObjects/sidePanel.page.js:82]: need a method to delete the phone number. side panel is present only when user profile is incomplete. so phone number must be deleted after each test, therefore I need a way to get there. would be grateful for any suggestions on how to do this better

File: pageObjects/sidePanel.page.js (hunk 1 above)
Comment (cyxoi1) [reply pageObjects/sidePanel.page.js:2]: for debugging only. to be removed

File: pageObjects/sidePanel.page.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
```
[the remark this answers is not in the document]
Comment (cyxoi1) [reply pageObjects/sidePanel.page.js:59]: to be removed

File: pageObjects/sidePanel.page.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (vasylynash) [reply pageObjects/sidePanel.page.js:80]: Create a separate PO for settingsPhone (we have settingsEmail and settingsPassword for now), add the needed methods and getters there, add the import in your files, and use them

**Pull request closed without merging**